db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/HentPersistertLøser.kt

```diff
@@ -39,23 +40,27 @@ class HentPersistertLøser(rapidsConnection: RapidsConnection, private val repos
     }
 
     override fun onBehov(packet: JsonMessage) {
-        val forespoerselId = packet[Key.FORESPOERSEL_ID.str].asText()
-        val transactionId = packet[Key.UUID.str].asText()
-        val event = packet[Key.EVENT_NAME.str].asText()
-        logger.info("Skal hente persistert inntektsmelding med forespørselId $forespoerselId")
-        sikkerLogger.info("Skal hente persistert inntektsmelding for pakke: ${packet.toJson()}")
-        try {
-            val dokument = repository.hentNyeste(forespoerselId)
-            if (dokument == null) {
-                logger.info("Fant IKKE persistert inntektsmelding for forespørselId $forespoerselId")
-            } else {
-                sikkerLogger.info("Fant persistert inntektsmelding: $dokument for forespørselId $forespoerselId")
+        measureTimeMillis {
```

Comment:
Her hadde det nok vært bedre å ha `measureTimeMillis` rundt kallet på funksjonen, inni `onPacket`.

Reply:
Denne koden blir fjernet i morgen. Jeg har ikke skrevet det for noe mer en rask debug.